content/Products/OpenshiftMonitoring/alerting.md

```diff
@@ -0,0 +1,30 @@
+# Alerting guidelines
+
+Please refer to the [Alerting Consistency](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md) OpenShift enhancement proposal for the recommendations.
+
+## Identifying alerting rules without a namespace label
+
+The enhancement proposal mentioned above states the following
+
+> Alerts SHOULD include a namespace label indicating the source of the alert.
+
+Unfortunately this isn't something that we can verify by static analysis. But we can still use the Telemetry data to identify OCP alerts that don't respect this statement.
+
+First, run this command against a live cluster to return the list of all OCP alert names:
```

Comment:
why do we need to fetch the alerts first? Couldn't we just check all alerts in `alerts` without a namespace, or use the cluster ID if interested in metrics of a particular cluster?

Reply:
because many users deploy their own alerting rules on top of the OCP built-in alerts (despite this being unsupported).

Reply:
I see, but here, we're fetching the alerts from `thanos-querier`, wouldn't it contain ALL firing alerts of the cluster?

Reply:
it's fetching the rules, not the alerts.

Reply:
I see now, you want this to be run on a "blank" cluster to only get the OCP built-in alerts.

If I understand correctly,  maybe we should clarify this.

(we may still have some false positives if a user overrides the alert definition (with one that doesn't produce a namespace) and keeps the same name, but this is not a problem).

Reply:
How about?
```suggestion
First, run this command against a blank cluster to only get the list of built-in OCP alert names:
```

(for me "live cluster" is a CU cluster)